test/app.test.js

```diff
@@ -16,7 +17,14 @@ process.env = {
 };
 const app = require('../app');
 
-const { KeyPair } = require('nearlib');
+const nearlib = require('nearlib');
+
+const NETWORK_ID = process.env.NETWORK_ID;
```

Comment:
hmm, is setting `NETWORK_ID` now a requirement for `yarn test` to work? I'd like us to avoid it.

Reply:
I wasn't sure what to do about this. You'll see that `NETWORK_ID` is also being used in app.js, which got introduced at the same time that I removed old RECOVERY_KEY stuff and the custom keyStore: 3073a99d. That's when I decided it probably made sense to add it to the ENV.

How would you update this?

Reply:
I think it generally isn't relevant in contract-helper context. It basically only matters when you have persistent key store (so that there can be different keys for different networks). In current contract-helper it would really work the same whether it is "default" or "bazinga" or "whatever", which means passing it externally is just extra hassle.